site/content/how-to/monitoring/tracing.md

```diff
@@ -70,6 +70,8 @@ To enable tracing, you must configure two resources:
 
 - `ObservabilityPolicy`: This resource is a [Direct PolicyAttachment](https://gateway-api.sigs.k8s.io/reference/policy-attachment/) that targets HTTPRoutes or GRPCRoutes. It is created by the [application developer](https://gateway-api.sigs.k8s.io/concepts/roles-and-personas/) and enables tracing for a specific route or routes. It requires the `NginxProxy` resource to exist in order to complete the tracing configuration.
```

Comment:
shall we also update https://deploy-preview-1884--nginx-gateway-fabric.netlify.app/nginx-gateway-fabric/how-to/configuration/control-plane-configuration/ similarly?

Reply:
Are you saying to link the API reference in that doc too?

Reply:
I think we can give the link and replace the spec section with a link to here https://deploy-preview-1884--nginx-gateway-fabric.netlify.app/nginx-gateway-fabric/reference/api/#gateway.nginx.org/v1alpha1.NginxGatewaySpec

Reply:
issue created https://github.com/nginxinc/nginx-gateway-fabric/issues/2119